tools/client/gsi-client.sh

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+#
+# collect Lustre client info for troubleshooting.  As of 1/8/2024 commands run to collect logs:
+	# uname -a
+	# cat /etc/lsb-release
+	# uptime; uptime -p
+	# netstat -rn
+	# netstat -Wan
+	# ifconfig -a
+	# printenv
+	# lfs --version
+	# lfs df -h
+	# lfs check all
+	# cd /var/log; tail -30 syslog
+	# cd /var/log; tar cvfz /tmp/client-gsi-2024-01-08T18:52:21/syslog.tgz syslog*
+	# sudo dmesg -T
+	# sudo sysctl -a
+	# sudo lnetctl stats show
+	# sudo lctl dl -t
+	# mount |egrep lustre; mount
+	# cat /etc/fstab |egrep lustre; cat /etc/fstab
+	# cat /sys/devices/virtual/bdi/lustrefs-ffffa09f150c6800/read_ahead_kb
+	# find /lustre -type f -print0 |xargs -0 -n 1 lfs  hsm_state
+
+usage() {
+    less <<EOF
+Usage ${0##*/} [options]
+collect lustre info
+    -l <log dir> starting dir for client-gsi-<date/time> dir.  Default to $HOME
+    -h usage
+EOF
+    exit
+}
+
+main() {
+    logdir=$HOME
+	while getopts "dhl:m:s:vD:M:Q:Z:" arg; do
```

Comment:
The opts seem more than the usage message above.

Reply:
@xyzisinus I have made the correction.  Can you take a look?